Lib/difflib.py

```diff
@@ -1134,32 +1138,37 @@ def unified_diff(a, b, fromfile='', tofile='', fromfiledate='',
      four
     """
 
+    if color and can_colorize():
+        t = get_theme(force_color=True).difflib
+    else:
+        t = get_theme(force_no_color=True).difflib
+
     _check_types(a, b, fromfile, tofile, fromfiledate, tofiledate, lineterm)
     started = False
     for group in SequenceMatcher(None,a,b).get_grouped_opcodes(n):
         if not started:
             started = True
             fromdate = '\t{}'.format(fromfiledate) if fromfiledate else ''
             todate = '\t{}'.format(tofiledate) if tofiledate else ''
-            yield '--- {}{}{}'.format(fromfile, fromdate, lineterm)
-            yield '+++ {}{}{}'.format(tofile, todate, lineterm)
+            yield '{}--- {}{}{}{}'.format(t.header, fromfile, fromdate, lineterm, t.reset)
+            yield '{}+++ {}{}{}{}'.format(t.header, tofile, todate, lineterm, t.reset)
```

Comment:
f-strings also keep us under the width limit
```suggestion
            yield f'{t.header}--- {fromfile}{fromdate}{lineterm}{t.reset}'
            yield f'{t.header}+++ {tofile}{todate}{lineterm}{t.reset}'
```

Reply:
Updated. For context, my thought was to keep the original style.